pages/_app.js

```diff
@@ -30,14 +31,16 @@ export default class RoostApp extends App {
     return (
       <React.Fragment>
         <Head>
-          <title>My page</title>
+          <link rel="icon" href="/favicon.ico" />
+          <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" />
           <meta
             name="viewport"
             content="minimum-scale=1, initial-scale=1, width=device-width" />
         </Head>
         <ThemeProvider theme={theme}>
           {/* CssBaseline kickstart an elegant, consistent, and simple baseline to build upon. */}
           <CssBaseline />
+          <Nav />
```

Comment:
What's this bad boy doing?

Reply:
Per convo with @lshadler - this is applying the new `<Nav />` component to all pages within `pages/`.